### ISSUE TITLE
[Suggestion] allow custom emotes for reacting

### ISSUE THREAD

**Issue by thisaray** [Juby210/text-react#2]
since you cant use for example 3 "r"s in one sentence, id would probably be helpful being able to set custom emotes as letters!

**Comment by DracoClaw** [comment]
yes! for example, i have a set of red letters c-z as there is already a and b in red. id love to be able to use these as well

**Comment by DracoClaw** [comment]
Just bringing this up again. Would be an amazing feature.

**Comment by Groovin-Dev** [comment]
This would be a really nice feature!

**Comment by thisaray** [comment]
Hi!, i am just going to ping it again!

**Comment by Errorcrafter** [comment]
99% sure this would be impossible, the api requests to get the emoji names could get yuo ratelimited

**Comment by thisaray** [comment]
@Errorcrafter could be solving by letting the player input specific emote IDs in the config

**Comment by MulverineX** [comment]
Or by reusing the emoji selector

**Comment by ArjixWasTaken** [comment]
Uhh, it's really easy to add emotes on your own, just fork the plugin and modify the emotes object.
